internal/cmd/destinations.go

```diff
@@ -5,6 +5,7 @@ import (
 	"fmt"
 
 	"github.com/spf13/cobra"
+	"sigs.k8s.io/yaml"
```

Comment:
For anyone else curious about this. It turns out we have to use `sigs.k8s.io/yaml` so that the field names come from the `json` tags. If we used the more standard yaml library we'd have to add `yaml` names to all the fields, and the output would look strange for some fields (ex: would use `{}` for the zero value of `api.Time` instead of `null`).

Although the second part of that we could fix by changing to `MarshalText` instead of `MashalJson`.